test/src/api/commands/client/testSetNetworkConditions.js

```diff
@@ -53,8 +62,11 @@ describe('.setNetworkConditions()', function () {
       },
       function (result) {
         assert.deepStrictEqual(result.value, null);
-      }
-      );
+        assert.strictEqual(expected.download_throughput, 460800);
```

Comment:
it's better to move these assertions in the callback to `client.start()`, right before calling `done()`.

Reply:
we should modify all the other tests too, if they're using a similar approach.